nixos/modules/services/web-apps/glitchtip.nix

```diff
@@ -0,0 +1,293 @@
+        description = "The address to listen on.";
+        default = "127.0.0.1";
+        example = "0.0.0.0";
+      };
+
+      port = lib.mkOption {
+        type = lib.types.port;
+        description = "The port to listen on.";
+        default = 8000;
+      };
+
+      settings = lib.mkOption {
+        description = ''
+          Configuration of GlitchTip. See <https://glitchtip.com/documentation/install#configuration> for more information.
+        '';
+        default = { };
+        defaultText = lib.literalExpression ''
+          {
+            DEBUG = 0;
+            DEBUG_TOOLBAR = 0;
+            DATABASE_URL = lib.mkIf config.services.glitchtip.database.createLocally "postgresql://@/glitchtip";
+            REDIS_URL = lib.mkIf config.services.glitchtip.redis.createLocally "unix://''${config.services.redis.servers.glitchtip.unixSocket}";
+            CELERY_BROKER_URL = lib.mkIf config.services.glitchtip.redis.createLocally "redis+socket://''${config.services.redis.servers.glitchtip.unixSocket}";
+          }
+        '';
+        example = {
+          GLITCHTIP_DOMAIN = "https://glitchtip.example.com";
+          DATABASE_URL = "postgres://postgres:postgres@postgres/postgres";
+        };
+
+        type = lib.types.submodule {
+          freeformType =
+            with lib.types;
+            attrsOf (oneOf [
+              str
+              int
+              bool
+            ]);
+
+          options = {
+            GLITCHTIP_DOMAIN = lib.mkOption {
+              type = lib.types.str;
+              description = "The URL under which GlitchTip is externally reachable.";
+              example = "https://glitchtip.example.com";
+            };
+
+            ENABLE_USER_REGISTRATION = lib.mkOption {
+              type = lib.types.bool;
+              description = ''
+                When true, any user will be able to register. When false, user self-signup is disabled after the first user is registered. Subsequent users must be created by a superuser on the backend and organization invitations may only be sent to existing users.
+              '';
+              default = false;
+            };
+
+            ENABLE_ORGANIZATION_CREATION = lib.mkOption {
+              type = lib.types.bool;
+              description = ''
+                When false, only superusers will be able to create new organizations after the first. When true, any user can create a new organization.
+              '';
+              default = false;
+            };
+          };
+        };
+      };
+
+      environmentFiles = lib.mkOption {
+        type = lib.types.listOf lib.types.path;
+        default = [ ];
+        example = [ "/run/secrets/glitchtip.env" ];
+        description = ''
+          Files to load environment variables from in addition to [](#opt-services.glitchtip.settings). This is useful to avoid putting secrets into the nix store. See <https://glitchtip.com/documentation/install#configuration> for more information.
```

Comment:
```suggestion
          Files to load environment variables from in addition to [](#opt-services.glitchtip.settings).
          This is useful to avoid putting secrets into the nix store.
          See <https://glitchtip.com/documentation/install#configuration> for more information.
```